### PR TITLE
updates

### PR DESCRIPTION
**Pull request by n-hutton**
see comments

### CODE REVIEW

File: constellation/requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 certifi==2018.4.16
 chardet==3.0.4
 idna==2.7
-requests==2.19.1
+requests==2.20
```
Comment (n-hutton) [inline constellation/requirements.txt]: git was complaining 2.19 was insecure

File: fetch-ledger-develop-image/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,7 @@
 FROM ubuntu:18.04
 
+RUN apt-get update
```
Comment (n-hutton) [inline fetch-ledger-develop-image/Dockerfile]: docker build fails for me without this, when trying to apt-get install vim and git

File: fetch-ledger-develop-image/Dockerfile [hunk 3]
```diff
@@ -30,6 +32,7 @@ RUN HOME=/home/default && \
 
 ENV CC=/usr/bin/clang
 ENV CXX=/usr/bin/clang++
+ENV CCACHE_DIR=/build/docker_ccache
```
Comment (n-hutton) [inline fetch-ledger-develop-image/Dockerfile]: This is useful for me to have persistent caching. Note ccache is not installed in the container by default

**Comment by pbukva** [comment]
CI build SUCCESSFUL.

File: constellation/requirements.txt (hunk 1 above)
Comment (ejfitzgerald) [reply constellation/requirements.txt]: Yep I saw that.

**Pull request merged**